windows/deploy_windows.ps1

```diff
@@ -256,6 +257,19 @@ function Build-App-Variants
 # Build Windows installer
 Function Build-Installer
 {
+    param(
```

Comment:
```suggestion
    param (
```

Style

Reply:
Only 1 other "param" in the scripts is following this coding style (with the space), all others user param(.
There are also two using Param instead of param. 
Would it be better to update all to use the style "Param (" ?  Or "param ("?  
not sure what the preference would be for the of the first capital...if you let me know I can amend all and align them.

Reply:
Hmm. So I didn’t check that. Shame on me ;-). Personally I would prefer param( but that’s not the jamulus C++ style.

Reply:
Okay will make them all matching and use param( for this powershell script....